tests/test_jx/test_set_ops.py

```diff
@@ -1098,16 +1099,16 @@ def test_select_nested_column(self):
             "expecting_list": {
                 "meta": {"format": "list"},
                 "data": [
-                    [{"b": 1, "c": 1}, {"b": 2, "c": 1}],
-                    [{"b": 1, "c": 2}, {"b": 2, "c": 2}]
-                ]
+                [{"b": 1, "c": 1}, {"b": 2, "c": 1}],
+                [{"b": 1, "c": 2}, {"b": 2, "c": 2}]
+            ]
```

Comment:
For next time: Can you change your whitespace settings so that the array contents are indented?